unblob/handlers/archive/zip.py

```diff
@@ -58,54 +65,49 @@ class ZIPHandler(StructHandler):
     """
     HEADER_STRUCT = "local_file_header_t"
```

Comment:
cd_file_header_t

Reply:
I switched to `end_of_central_directory_t` as it's the header content that's the most interesting to someone running unblob in debug mode (with colored struct being printed out when we call `self.parse_header`).